stable/fluentd/templates/deployment.yaml

```diff
@@ -98,6 +101,7 @@ spec:
         - name: config-volume-{{ template "fluentd.fullname" . }}
           configMap:
             name: {{ template "fluentd.fullname" . }}
+            defaultMode: 0777
```

Comment:
Why do we need this change?

Reply:
Otherwise the shell script cannot be run. We can add a specific mount if you'd prefer.